lighty-netconf-device/src/main/java/io/lighty/netconf/device/requests/EditConfigRequestProcessor.java

```diff
@@ -295,7 +300,7 @@ private static YangInstanceIdentifier getYangInstanceIdentifier(final List<QName
         while (iterator.hasNext()) {
             final QName currentQname = parseQname(effectiveModelContext, iterator.next());
             contextNode = contextNode.getChild(currentQname);
-            while (contextNode.isMixin()) {
+            while (requireNonNull(contextNode).isMixin()) {
```

Comment:
Extract `requireNonNull` outside of  `while` statement

Reply:
`contextNode = contextNode.getChild(currentQname)` on line 306 is `@Nullable`
https://github.com/PANTHEONtech/lighty-netconf-simulator/pull/63/files/d16b57b9d294315a7bd9c5c51659f92c96f921f7#diff-cae6f49f2f2e0c99726ccad0e34c4a7c3eac7104507222777166ded9dcbd55a5R306
we should check this every time

Reply:
not null checked on both assignments